resources/views/layout/public.blade.php

```diff
@@ -24,6 +24,11 @@
 
     <script src='https://api.mapbox.com/mapbox-gl-js/v3.4.0/mapbox-gl.js'></script>
     <link href='https://api.mapbox.com/mapbox-gl-js/v3.4.0/mapbox-gl.css' rel='stylesheet'/>
+    <script
+        defer src="https://analytics.blumilk.pl/script.js"
+        data-website-id="f07919bb-dcef-4970-9efb-e732177147ab"
+        data-domains="dev.website.blumilk.pl">
```

Comment:
It will be changed for production related data, after tests on beta.

Reply:
Do we need to collect data from beta? I thought that after tests and umami configuration, we will only use it on production domain (blumilk.pl).

Reply:
What about local instances? Would it report any usage with this configuration?

Reply:
No, it won't. It will report only if domain is `dev.website.blumilk.pl` (at this moment). 

it's configured here:
```
data-domains="dev.website.blumilk.pl"
```
We can add more domains separated by commas if needed.